src/pudl/analysis/allocate_net_gen.py

```diff
@@ -146,25 +176,14 @@ def allocate_gen_fuel_by_gen(pudl_out):
         ]
         + list(pudl_out.gens_eia860().filter(like="energy_source_code")),
     ]
-
     # do the allocation! (this function coordinates the bulk of the work in
     # this module)
-    gen_pm_fuel = allocate_gen_fuel_by_gen_pm_fuel(gf, gen, gens)
-    # aggregate the gen/pm/fuel records back to generator records
-    gen_allocated = agg_by_generator(gen_pm_fuel)
-    _test_gen_fuel_allocation(gen, gen_allocated)
-
-    # make the output mirror the gen_original_eia923()
-    gen_allocated = pudl.output.eia923.denorm_generation_eia923(
-        g_df=gen_allocated,
-        pudl_engine=pudl_out.pudl_engine,
-        start_date=pudl_out.start_date,
-        end_date=pudl_out.end_date,
-    )
-    return gen_allocated
+    gen_pm_fuel = allocate_gen_fuel_by_gen_pm_fuel_guts(gf, gen, gens)
+    _test_gen_fuel_allocation(gen, gen_pm_fuel)
+    return gen_pm_fuel
 
 
-def allocate_gen_fuel_by_gen_pm_fuel(gf, gen, gens, drop_interim_cols=True):
+def allocate_gen_fuel_by_gen_pm_fuel_guts(gf, gen, gens, drop_interim_cols=True):
```

Comment:
I obviously need to rename this.... but I used it's previous name for the wrapper function that grabs the pudl_out tables and then does this stuff. suggestions?